website/docs/docs/building-a-dbt-project/building-models/configuring-incremental-models.md

```diff
@@ -142,7 +142,41 @@ On warehouses that do not support `merge` statements, a merge is implemented by
 Transaction management is used to ensure this is executed as a single unit of work.
 
 ## What if the columns of my incremental model change?
-If you add a column from your incremental model, and execute a `dbt run`, this column will _not_ appear in your target table.
+
+:::tip New `on_schema_change` config in dbt version `v0.21.0`
+
+Incremental models can now be configured to include an optional `on_schema_change` parameter to enable additional control when incremental model columns change. These options enable dbt to continue running incremental models in the presence of schema changes, resulting in fewer `--full-refresh` scenarios and saving query costs.  
+
+:::
+
+You can configure the `on_schema_change` setting as follows.
+
+<File name='models/staging/fct_daily_active_users.sql'>
+
+```sql
+{{
+    config(
+        materialized='incremental',
+        unique_key='date_day',
+        on_schema_change=['ignore', 'fail', 'append_new_columns', 'sync_all_columns'] --choose one
+    )
+}}
+```
+
+</File>
```

Comment:
Could you also include an example of setting this from the project file? Similar to how we show both for `incremental_strategy` below:
```yaml
models:
  +on_schema_change: sync_all_columns
```